scopes/component/ui/version-block/change-log.data.ts

```diff
@@ -27,16 +29,3 @@ export type Version = {
   message: string;
   contributors: AccountObj[];
 };
-
-export type AccountObj = {
-  name?: string;
-  accountType?: AccountTypes;
-  displayName?: string;
-  profileImage?: string;
-};
-
-export enum AccountTypes {
-  org = 'organization',
-  user = 'user',
-  default = 'default',
-}
```

Comment:
Did you remove this? Or it's come from another commit?

Reply:
I removed this - the exact same type + enum had been declared in two separate places in the code. I removed this one and made the avatar one the source of truth as it makes more sense there (than in a version block, anyway). 
Ideally there would be an 'Account' component which would define these account exports, but Avatar's as close as we currently have for that

Reply:
I think @odedre or @KutnerUri need to review this.

Reply:
@KutnerUri @odedre - can one of you please review this and the other file I tagged you in below

Reply:
sounds good to me, they are the same and `contributors` are probably used in an Avatar component.
Maybe we should extract AccountObj from Avatar? Not sure where to put them

Reply:
Also wasnt sure where else to put the objects, so left them in Avatar for now as it's the best option available. 
Like in my comment above maybe there should be an Account aspect/component which manages account related objects (and data?)